README.md

```diff
@@ -1 +1,54 @@
-# operate-first-workloads
\ No newline at end of file
+### operate first workloads
+
+#### This readme contains a getting started guide to Operate First.
+
+If you are interested in bringing your project to the Operate First Environment at the Mass Open Cloud (MOC) you have come to the right place. Below is important information and instructions on how to get started.
+
+### There are 3 parts to Operating First
+
+1. Operate First is a concept of bringing pre-release open source software to a production cloud environment.
+
+2. The Open Infrastructure Labs is the upstream where the cloud operators and project developers work in the open to Operate First.
+
+3. The Mass Open Cloud (MOC) is the production cloud resource where projects are run.
+
+### First off - understand the environment
```

Comment:
I wouldn't put this content in this document (or at least, not at this location). Unlike the rest of this README, which is all about who we are, how to communicate with us, etc, this is a set of strictures on *how* groups can participate with us.

I think we should have a separate "so you want to bring your project to operate first" document that includes this material.

If not that, I would least move this down below the "Bring your project" heading, and probably soften the opening sentence.

Reply:
I like the idea of a separate doc for this. Given the comments and / or misunderstanding about the environment it is key information. I will move it down as you suggest, and it can be replaced with a link to a separate doc when one is created.

Reply:
We could also use the content in https://github.com/open-infrastructure-labs/ops-issues/issues/32 if you like that better, quoted here:

This applies to the operate first environment in general.

test clusters
can be destroyed by the requesting party at any time
requesters can grant access to collaborators
can be destroyed by operate first ops with X days prior notice
production clusters
can go down anytime due to software failures
no backup guaranty for data - but you can work with ops to implement a strategy for your workload
scheduled outages are communicated with X days prior notice
support is best-effort in a community fashion - no SLAs
operations are practiced in a state of the art SRE fashion, aiming for maximum transparency

Reply:
I did soften the intro...thanks for all the input!